server/aws-lsp-codewhisperer/src/shared/amazonQServer.ts

```diff
@@ -35,6 +38,16 @@ export const AmazonQServiceServerFactory =
                 sdkInitializator,
             })
 
+            // Initialize ATX Token Service Manager for ATX FES support
+            AtxTokenServiceManager.initInstance({
+                credentialsProvider,
+                lsp,
+                workspace,
+                logging,
+                runtime,
+                sdkInitializator,
+            })
```

Comment:
same comment from last PR -- should this be in its own server factory or is too intertwined with the existing q server